libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Network/Network/Queue/CSFNetwork.m

```diff
@@ -93,7 +93,7 @@ + (instancetype)currentNetwork {
 
 + (instancetype)networkForUserAccount:(SFUserAccount*)account {
     CSFNetwork *instance = nil;
-    if (!account.isTemporaryUser) {
+    if (account!=nil) {
```

Comment:
Actually, it wouldn't necessarily return `nil`, as `CSFNetworkInstanceKey()` in that case is going to return the string "(null)-(null)-(null)", which could still be a valid key for a "non-user" network instance.  So I think that could be fine, at least on its surface.  We'll have to see how it plays out downstream.